tests/test_ses/test_server.py

```diff
@@ -13,3 +16,16 @@ def test_ses_list_identities():
 
     res = test_client.get("/?Action=ListIdentities")
     res.data.should.contain(b"ListIdentitiesResponse")
+
+
+def test_ses_get_send_statistics():
+    backend = server.create_backend_app("ses")
+    test_client = backend.test_client()
+
+    res = test_client.get("/?Action=GetSendStatistics")
+    res.data.should.contain(b"GetSendStatisticsResponse")
+
+    # Timestamps must be in ISO 8601 format
+    groups = re.search("<Timestamp>(.*)</Timestamp>", res.data.decode("utf-8"))
+    timestamp = groups.groups()[0]
+    datetime.fromisoformat(timestamp)
```

Comment:
Looks like the tests are still failing because the `fromisoformat` is only available as of Python 3.7, and we're still supporting 3.6 - you'll have to find an alternative for this method.

Reply:
Switched to 3.6 compatible `strptime` in [0591514](https://github.com/spulec/moto/pull/4826/commits/05915141b7d619873448afdda7087793838ba776)